sympy/stats/tests/test_discrete_rv.py

```diff
@@ -86,3 +88,16 @@ def test_conditional():
     assert P(X > 3, X > 2) == S(1)/3
     assert P(Y > 2, Y < 2) == 0
     assert P(Eq(Y, 3), Y >= 0) == 9*exp(-3)/2
+
+def test_product_spaces():
+    X1 = Geometric('X1', S(1)/2)
+    X2 = Geometric('X2', S(1)/3)
+    from sympy import symbols
+    n = symbols('n')
+    oo = S.Infinity
+    assert str(P(X1 + X2 < 3, evaluate=False) == Sum(Piecewise((2**(X2 - n - 2)*(2/3)**(X2 - 1)/6,
+    (-X2 + n + 3 >= 1) & (-X2 + n + 3 < oo)), (0, True)), (X2, 1, oo), (n, -oo, -1)))
+    assert str(P(X1 + X2 > 3) == Sum(Piecewise((2**(X2 - n - 2)*(2/3)**(X2 - 1)/6,
+        (-X2 + n + 3 >= 1) & (-X2 + n + 3 < oo)), (0, True)), (X2, 1, oo), (n, 1, oo)))
+    assert str(P(Eq(X1 + X2, 3)) == Sum(
+        Piecewise((2**(X2 - 2)*(S(2)/3)**(X2 - 1)/6, X2 <= 2), (0, True)), (X2, 1, oo)))
```

Comment:
I was checking this result with Mathematica, I get:
```
In[12]:= Probability[x+y==3, {x\[Distributed]GeometricDistribution[1/2], y\[Distributed] GeometricDistribution[1/3]}]
         175
Out[12]= ----
         1296
```
I think your expression is equal to `7/36`

Reply:
Sorry, Mathematica and SymPy have slightly different definitions of the PDF.